src/components/Versions/styles.ts

```diff
@@ -14,6 +14,7 @@ export const Spacer = styled('div')({
   borderBottom: '1px dotted rgba(0, 0, 0, 0.2)',
   whiteSpace: 'nowrap',
   height: '0.5em',
+  margin: '0 16px',
```

Comment:
Could you give more context about the margin?

Reply:
I noticed that there was no space between the Tabs component and the content, so I added 16px margin. Maybe in the future we may have some common spaces, making the application more homogeneous.